ext/calendar/calendar.stub.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/** @return int|false */
+function cal_days_in_month(int $calendar, int $month, int $year) {}
+
+/** @return array<string, array<int, string>&int&string>|false */
+function cal_from_jd(int $jd, int $calendar) {}
+
+/** @return array<string&int>|false */
+function cal_info(?int $calendar = UNKNOWN) {}
+
+/** @return int|false */
+function cal_to_jd(int $calendar, int $month, int $day, int $year) {}
+
+function easter_date(int $year = UNKNOWN): int {}
```

Comment:
Looks like technically this also accepts the `$method` parameter, which doesn't seem relevant to this function. I'm fine with omitting it here.

Reply:
Kk, the same seems to be about `easter_day`

Reply:
The $method parameter is relevant even for easter_date, e.g.
````php
var_dump(easter_date(2019)); // int(1555797600)
var_dump(easter_date(2019, CAL_EASTER_ALWAYS_JULIAN)); // int(1555279200)
````
While calculating an easter date between 1970 and 2038 according to the Julian calendar is rather uncommon, it might be useful in some cases. I suggest to document the $method parameter of easter_date(), and to add it to the stub.